### PR TITLE
Updates running.org doc with latest instructions 

### PR DESCRIPTION
**Pull request by code-chronos**
# Description

- Clear instructions on how to run the epic-server, epic-wallet binary files. 
- Moving foundation.json file and updates to epic-server.toml file.

### CODE REVIEW

File: doc/running.org [hunk 1]
```diff
@@ -160,27 +175,28 @@ The following steps explain how to install and run the epic-wallet:
 
 1. Download the latest version of the *epic-wallet* package [[https://epic.tech/downloads/][here]].
 
-2. Open a new terminal window and navigate to the directory where you
-   downloaded the .deb file and execute the following command changing
-   the argument <your-epic-wallet.deb> to the name of your downloaded
-   file:
+2.  Unzip the downloaded epic wallet package. A binary file is available inside the unzipped paxkage.
```
Comment (blacktyger) [inline doc/running.org:178]: typo `paxkage`

File: doc/running.org [hunk 2]
```diff
@@ -67,26 +67,41 @@ The following steps explain how to install and run the epic server:
 
 1. Download the latest version of the *epic* package [[https://epic.tech/downloads/][here]].
 
-2. In the terminal, navigate to the directory where you downloaded the
-   .deb file and execute the following command changing the argument
-   <your-epic-server.deb> to the name of your downloaded file:
+2.  Unzip the downloaded epic server package. A binary file and a foundation.json file are available inside the unzipped paxkage.
```
Comment (blacktyger) [inline doc/running.org:70]: typo `paxkage`

**Review by blacktyger — CHANGES_REQUESTED** [review]
Fix the 2 typo's

File: doc/running.org (hunk 1 above)
Comment (code-chronos) [reply doc/running.org]: Thanks. Updated

**Review by blacktyger — APPROVED** [review]

**Review by johanneshahn — APPROVED** [review]

**Pull request merged**